draft-ietf-dnsop-svcb-https.md

```diff
@@ -535,6 +535,10 @@ such as Happy Eyeballs {{!HappyEyeballsV2=RFC8305}}.
 Some important optimizations are discussed in {{optimizations}}
 to avoid additional latency in comparison to ordinary AAAA/A lookups.
 
+Although SVCB can only provide alternative endpoints for any pre-existing
+protocol, new protocols could be defined that require a ServiceMode record with no fallback to AAAA/A address records.
+Such "SVCB-reliant" protocols SHALL terminate upon reaching step 5.
```

Comment:
```suggestion
Such "SVCB-reliant" protocols SHALL:
* ... start off step 1 with ADDR_QNAME being undefined
* ... not issue A/AAAA queries when ADDR_QNAME is undefined
* ... terminate upon reaching step 5
```

Reply:
Yes, I think that would do.  Though, as I noted in mail, it would be better if the current optimization were framed as such and the conditions under which the optimization apply more clearly articulated.

Reply:
I've drawn up #291 as a _third_ alternative solution here, which implements that reframing.  I'm not sure it's more comprehensible than this text, but it is probably more precise.  Please review.